portfolio/src/main/webapp/background.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" content="width=device-width, initial-scale=1"> 
+    <title>Background</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body id="sub-page">
+    <div class="sidenav">
+      <a href="index.html">Home</a>
+      <a href="background.html">Background</a>
+      <a href="projects.html">Projects</a>
+      <a href="personal.html">Personal</a>
+      <a href= "https://www.linkedin.com/in/theodora-mara-pislar-983117196/"#
+          target="_blank">LinkedIn</a>
+      <a href="https://github.com/maraPislar" target="_blank">Github</a>
+    </div>
+    
+    <div class="main">
+      <div class="content">
+        <h1>Background</h1>
+        <h4 class="title">Where I live</h4>
+        <p>
+            My home town is called Radauti.
+            Radauti is a small town from the North of Romania and is
+            very popular in the country because of a soup called "
+            Ciorba Radauteana". This soup takes over its name because it 
+            was invented by an old lady in town. Now it is served in 
+            most of the restaurants all around the country.
+        </p>
+
+        <img src="/images/radauti2.jpg" class="some-image">
+
+        <h4 class="title">Family</h4>
+        <p>
+            I was born on the 25th of July in 2000 in a loving family.
+            We love to go on trips together, especially hiking.
+            I have two other sisters, me being the middle child. Miruna
+            is 24 and Sofia is 5 at the moment. It's very hard to
+            take a clear picture with Sofia, she is not 
+            very patient.<br><br>
+            Fun fact: there were many times in my life when
+            people thought me and Miruna were twins, when in fact we are
+            4 years apart.
+        </p>
+
+        <img src="/images/family.jpg" class="some-image">
+
+        <h4 class="title">Education</h4>
+        <p>
+          <ul>
+            <h4>2015-2019</h4>
+            <li>"Eudoxiu Hurmuzachi" College, Radauti</li>
+            <li>Mathematics and Informatics profile</li>
+            <li>Learned the basics in Informatics in C++</li>
+            <h4>2019-2022</h4>
+            <li>University of Southampton, United Kingdom</li>
+            <li>BSc Computer Science</li>
+
+            <h4>Languages:</h4>
+            <li>Romanian (native)</li>
+            <li>English</li>
+            <li>German</li>
+
+            <h4>Programming Languages:</h4>
+            <li>Java, C++, Go, JavaScript, Python </li>
+            <li>HTML and CSS for web development</li>
+          </ul>
+
+          <img src="/images/school.jpg" class="some-image">
+          <br><br><br><br>
```

Comment:
`<br>...<br>` this feels like presentation, something better be dealt with in CSS